### PR TITLE
improve: remove photon check

### PR DESCRIPTION
**Pull request by jasonkuhrt**
It's time.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -6903,9 +6903,9 @@ unc-path-regex@^0.1.2:
   resolved "https://registry.yarnpkg.com/unc-path-regex/-/unc-path-regex-0.1.2.tgz#e73dd3d7b0d7c5ed86fbac6b0ae7d8c6a69d50fa"
   integrity sha1-5z3T17DXxe2G+6xrCufYxqadUPo=
 
-"undici@github:mcollina/undici":
-  version "0.5.0"
-  resolved "https://codeload.github.com/mcollina/undici/tar.gz/ed672a39ee63f4c5fa13d0c826a0d47edf77e9fa"
+undici@mcollina/undici:
```
Comment (jasonkuhrt) [inline yarn.lock:6906]: 🤷‍♂️

File: src/builder.ts [hunk 2]
```diff
@@ -183,17 +182,9 @@ let defaultClientPath: string
 if (process.env.NEXUS_PRISMA_CLIENT_PATH) {
   defaultClientPath = process.env.NEXUS_PRISMA_CLIENT_PATH
 } else if (process.env.NEXUS_PRISMA_LINK) {
-  defaultClientPath = path.join(process.cwd(), '/node_modules/@prisma/photon')
```
Comment (jasonkuhrt) [inline src/builder.ts:186]: autoformatting...

File: yarn.lock (hunk 1 above)
Comment (Weakky) [reply yarn.lock:6906]: Yeah saw that recently too, I have no idea where it comes from

**Review by Weakky — APPROVED** [review]
LGTM 🚀

**Pull request merged**